db/seeds.rb

```diff
@@ -1,7 +1,418 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require 'open-uri'
+
+Referral.destroy_all
+Tie.destroy_all
+UserInterest.destroy_all
+Interest.destroy_all
+User.destroy_all
+
+user1 = User.new(username: "mary", email: "mary@user.com", first_name: "Mary", last_name: "Smith", password: "123456",
+  bio: "Hi peeps, I am Mary! I'm loving the big city life and always keen to meet new like minded people.",
+  summary: "I love crossfit, fashion and crypto", status: "Wanna join me for a crossfit sesh?")
+file = URI.open('https://images.unsplash.com/photo-1520024146169-3240400354ae?ixid=MnwxMjA3fDB8MHxzZWFyY2h8NHx8Y29vbCUyMGdpcmxzfGVufDB8fDB8fA%3D%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=900&q=60')
+user1.photos.attach(io: file, filename: 'photo.jpg', content_type: 'image/jpg')
```

Comment:
Could we have two or three photos attached for user 1 & 2. The additional photos doesn't even have to be a face, can be a house, a car, boat, or something they could own

Reply:
Yes can add a couple more for them.